examples/StrongCustomerAuthenticationExample.php

```diff
@@ -162,8 +163,9 @@
  * Billing object
  * Delivery (or Billing object again, if you want to have the delivery address the same as the billing address)
  * User object
+ * Api version - by default is used 'v3' for ALU v3
  */
-$request = new Request($cfg, $order, $billing, $delivery, $user);
+$request = new Request($cfg, $order, $billing, $delivery, $user, AluV3::API_VERSION_V3);
```

Comment:
Shouldn't the default be alu V4? This would encourage new user to use the new version

Reply:
We will set the default version to v4 in another PR, here is not implemented that version :D